### DISCUSSION TITLE
Support for Reverse Connection

### DISCUSSION THREAD

**Discussion by cpipero** [FreeOpcUa/python-opcua#1314] [Q&A]
I could not find any information on Reverse Connect (either on existing features or future roadmap) and was wondering if there is any plan to support.

See OPC UA Specs at https://reference.opcfoundation.org/v104/Core/docs/Part7/6.6.5/ .

I'm planning to implement a client but the customer is mandating reverse connect to their servers for firewall constraints.

Thanks for any information about this feature.

**Comment by AndreasHeine** [comment]
dont think so see:

README.md
> The library is in maintenance mode. The plan is now to focus on the asyncio version of this library: opcua-asyncio which also has a sync-wrapper, with very few changes in API
